### PR TITLE
164399894 Fix data returned using data range in `analyticsForDailyRoomEvents` query

### PR DESCRIPTION
**Pull request by RonKbS**
#### Description
Currently, the data being returned by the application using that particular query does not exclude all data for a specified date range. The changes introduced here rectify this.

#### Type of change
Bug fix (non-breaking change which fixes an issue)

#### How Has This Been Tested?
To recreate the bug;
 - Create a room with the calenderId posted on this [thread](https://andela.slack.com/archives/GAJJA2ETB/p1551788890003800).
- Run the query below on the develop branch;
```
query{
    analyticsForDailyRoomEvents(startDate:"jan 10 2019", endDate:"feb 15 2019"){
        day
        events{
            eventSummary
            startTime
            endTime
            roomName
            noOfParticipants
        }
    }
}
```
- The response returned will have data not part of the date range if one is to copy and paste it into the pages app and then search for 2018.
- Check out to the branch on which this PR has been raised.
- Rerunning the query will show that the data has been excluded

#### Any background context you want to provide?
To increase coverage, the events.json file will need to be adjusted such that data that matches what is being filtered out by these changes is included.

#### Checklist:
- [x] My code follows the style guidelines of this project

- [x] I have linted my code prior to submission

- [x] I have made the corresponding changes to the documentation

- [ ] My changes generate no new warnings 

- [ ] I have added tests that prove my feature works

- [x] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[#164399894](https://www.pivotaltracker.com/story/show/164399894)

### CODE REVIEW

**Comment by ThaDeveloper** [comment]
Kindly work on the codeclimate issues.

**Comment by RonKbS** [comment]
> Kindly work on the codeclimate issues.

Kindly look at the context in the PR description regarding this.

File: helpers/calendar/analytics.py [hunk 1]
```diff
@@ -36,6 +36,9 @@ def get_events_number_meetings_room_analytics(self, query, start_date, end_date)
                 number_of_meetings.append(0)
             else:
                 for event in all_events:
+                    if event["start"]["dateTime"] < start_date:
```
Comment (nakatuddesuzan) [inline helpers/calendar/analytics.py]: Hello @RonKbS , great work.  About the 3 similar blocks of code, how about you use a function that can be called.?

Comment (RonKbS) [reply helpers/calendar/analytics.py]: > Hello @RonKbS , great work. About the 3 similar blocks of code, how about you use a function that can be called.?

This has been worked on @nakatuddesuzan.

**Comment by dnuwa** [comment]
> > Kindly work on the codeclimate issues.
> 
> Kindly look at the context in the PR description regarding this.

I trust that the DevOps team needs to be contacted on the same. kindly update us in this @RonKbS thanx.

**Comment by RonKbS** [comment]
> Fix the merge conflicts

The conflicts have been fixed.

**Pull request closed without merging**